docs/velox-backend-support-progress.md

```diff
@@ -299,20 +299,19 @@ Gluten supports 199 functions. (Drag to right to see all data types)
-| array_union                  |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| arrays_overlap               | array_overlap          | S                     |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
+| array_union                  | array_union            | array_union           | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
+| arrays_overlap               | array_overlap          | array_overlap         | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | arrays_zip                   | zip                    |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | cardinality                  | cardinality            |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
 | element_at                   | element_at             | element_at            | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          | S     | S   |        |     |
 | exists                       | any_match              |                       | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| explode, explode_outer       |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
-| explode_outer, explode       |                        |                       |        |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
+| explode, explode_outer       | explode                | explode               | S      |                          |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |     |        |     |
```

Comment:
explode is supported, while explode_outer is not. [This PR](https://github.com/apache/incubator-gluten/pull/8333) is working on it.